torchbenchmark/models/resnet50/__init__.py

```diff
@@ -43,6 +43,13 @@ def __init__(self, device=None, jit=False, train_bs=32, eval_bs=32, extra_args=[
             self.eval_model.eval()
             self.eval_model = torch.jit.optimize_for_inference(self.eval_model)
 
+    def get_flops(self, test='eval'):
```

Comment:
We need to time the result by 2 due to how we count the FLOPS differently. In out context, one fused multiplication add (FMA) counts as 2 flops instead of one.

Reply:
Thanks. Addressed.